packages/blue-migration-sdk-viem/src/positions/supply/index.ts

```diff
@@ -0,0 +1,53 @@
+import type { Address, ChainId } from "@morpho-org/blue-sdk";
+
+import type { MigrationBundle } from "../../types/actions.js";
+import type {
+  MigratableProtocol,
+  SupplyMigrationLimiter,
+} from "../../types/index.js";
+
+export namespace MigratableSupplyPosition {
+  export interface Args {
+    amount: bigint;
+    minShares: bigint;
+    vault: Address;
+  }
+}
+
+export interface MigratableSupplyPositionConfig {
+  chainId: ChainId;
+  protocol: MigratableProtocol;
+  user: Address;
+  loanToken: Address;
+  supply: bigint;
+  supplyApy: number;
+  max: { value: bigint; limiter: SupplyMigrationLimiter };
+}
+
+export abstract class MigratableSupplyPosition
+  implements MigratableSupplyPositionConfig
+{
+  public readonly protocol: MigratableProtocol;
+  public readonly user: Address;
+  public readonly loanToken: Address;
+  public readonly supply: bigint;
+  public readonly supplyApy: number;
+  public readonly max: { value: bigint; limiter: SupplyMigrationLimiter };
+  public readonly chainId: ChainId;
```

Comment:
Ideally docs, but I honestly don't care for this sdk

Reply:
It's in my TODO

Reply:
https://linear.app/morpho-labs/issue/INTER-2094/add-doc-to-migration-sdk